tests/README.md

```diff
@@ -14,3 +14,35 @@ Advanced usage
 - XML results: `unish.phpunit.php --filter=testVersionString --log-junit results.xml`
 - Build the SUT and run test suite (slower) - `unish.clean.php`
 - Install the SUT in a given folder - `UNISH_TMP=/path/to/folder php unish.sut.php`
+
+Testing contrib module commands
+---------
+
+**Important:** The Unish tests should be placed under `tests/src/Unish` in your module. But if you still prefer other location, adjust the settings in `tests/phpunit.xml`.
+
+1. Create a copy of `tests/composer.json.dist` as `tests/composer.json`:
```

Comment:
Lets clarify that this is a copy of vendor/drush/drush/tests/composer.json.dist to the same dir, and not anything the module. Also, this is pretty unexpected. Its rare for module devs to be editing inside vendor dir. There is a danger of losing it on rebuild.

Reply:
Well, when I work on commands provided by modules, I'm opening Drush as project. I think this is an advantage because **I don't need a full Drupal install**. The single thing is that I need an external workcopy of the module. Then I'm testing the Drush commands from that module using this procedure.